### PR TITLE
[onert] Use workspace for execution data dump

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates TracingObserver and MinMaxRecorder to dump file into workspace.

Runtime config string `TRACE_FILEPATH` and `MINMAX_FILEPATH` also changes to `TRACING_MODE` and `MINMAX_DUMP` boolean.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: #13013

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

File: runtime/onert/api/nnfw/src/nnfw_api_internal.cc [hunk 1]
```diff
@@ -994,9 +994,9 @@ NNFW_STATUS nnfw_session::set_config(const char *key, const char *value)
 
   const std::string skey = key;
 
-  if (skey == config::TRACE_FILEPATH)
+  if (skey == config::TRACING_MODE)
   {
-    _coptions->trace_filepath = value;
+    _coptions->tracing_mode = value;
```
Comment (zetwhite) [inline runtime/onert/api/nnfw/src/nnfw_api_internal.cc:999]: `value` looks like `const char*`, but `_coptions->tracing_mode` looks like boolean. 
Is it okay and intended logic?

Comment (zetwhite) [reply runtime/onert/api/nnfw/src/nnfw_api_internal.cc:999]: otherwise looks good to me.

Comment (hseok-oh) [reply runtime/onert/api/nnfw/src/nnfw_api_internal.cc]: You're right. I'll fix this to set boolean value.

**Review by zetwhite — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**